docs/concepts/tokens/decentralized-exchange/index.md

```diff
@@ -4,17 +4,41 @@ parent: tokens.html
 metadata:
   indexPage: true
 seo:
-    description: The XRP Ledger contains a fully functional exchange where users can trade tokens for XRP or each other.
+    title: Decentralized Exchange (DEX)
+    description: Explore how the XRP Ledger powers its decentralized exchange (DEX) for seamless crypto trading. Learn how you can trade tokens securely on the XRP Ledger.
 targets:
   - en
 ---
-# Decentralized Exchange
+# Decentralized Exchange (DEX)
 
-The XRP Ledger has possibly the world's oldest _decentralized exchange_ (sometimes abbreviated "DEX"), operating continuously since the XRP Ledger's launch in 2012. The exchange allows users to buy and sell [tokens](../index.md) for XRP or other tokens, with minimal [fees](../../transactions/fees.md) charged to the network itself (not paid out to any party).
+The XRP Ledger has possibly the world's oldest _decentralized exchange_ (sometimes abbreviated "DEX"), operating continuously since the XRP Ledger's launch in 2012. The exchange allows users to buy and sell [tokens](../index.md) for XRP or other tokens, with minimal trading [fees](../../transactions/fees.md) charged to the network itself (not paid out to any party).
```

Comment:
This change is incorrect and should be reverted. The minimal fees that are charged to the network are not trading fees.